### ISSUE TITLE
Opening a file makes a new window in emacs 23

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ijt/emacs-nav#30]
```
What steps will reproduce the problem?
1. Open emacs 23
2. Open nav
3. Choose a file by hitting enter or clicking with the mouse.

What is the expected output? What do you see instead?
The file should open in the already-existing window on the right.
Instead a new window is created and the file opens there.

```

Original issue reported on code.google.com by `issac.tr...@gmail.com` on 10 Aug 2009 at 11:19

**Comment by GoogleCodeExporter** [comment]
```
This only seems to happen after manually resizing the frame.
```

Original comment by `issac.tr...@gmail.com` on 10 Aug 2009 at 11:25

**Comment by GoogleCodeExporter** [comment]
```
Fixed in dbac865dad.
```

Original comment by `issac.tr...@gmail.com` on 11 Aug 2009 at 8:04
- Changed state: **Fixed**
